.github/workflows/node.js.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        #node-version: [10.x, 12.x, 14.x]
+        node-version: [10.x, 14.x]
```

Comment:
12.x ????

Reply:
I can put it back. I waas afraid that minutes of Actions were billed. It's not the case.